src/mobx/stores/InfluenceVaultStore.ts

```diff
@@ -82,20 +82,34 @@ class InfluenceVaultStore {
           sett: vault.vaultToken.toLowerCase(),
         },
       });
-      const harvestConvertedSchedules = badgerTreeDistributions.map((e) => ({
-        token: e.token.id.startsWith('0x0x') ? e.token.id.slice(2) : e.token.id,
+      const { settHarvests } = await this.store.sdk.graph.loadSettHarvests({
```

Comment:
can you create a ticket for a follow up - maybe we could a utility method to the sdk for grabbing all harvests / distributions. 

https://github.com/Badger-Finance/badger-api/blob/main/src/accounts/accounts.utils.ts#L45-L80

something like this

Reply:
yeah ill put that and other refactor parts in a new ticket 👍